solutions/usaco-1037.mdx

```diff
@@ -14,6 +14,99 @@ By simulating handshakes for all values of $K$, we can find the minimum, maximum
 
 <LanguageSection>
 
+<CPPSection>
+
+```cpp
+#include <bits/stdc++.h>
+using namespace std;
+using ll = long long;
+using vi = vector<int>;
+#define all(x) begin(x), end(x)
+#define sz(x) (int)(x).size()
+
+void setIO(string name = "") {
+	ios_base::sync_with_stdio(0);
+	cin.tie(0);
+	if (sz(name)) {
+		freopen((name + ".in").c_str(), "r", stdin);
+		freopen((name + ".out").c_str(), "w", stdout);
+	}
+}
+
+struct Shake {
+	int t, x, y;
+	bool operator<(const Shake& tmp) { return t < tmp.t; }
+};
+
+int main() {
+	setIO("tracing");
+
+	int n, t;
+	string s;
+	cin >> n >> t >> s;
+
+	vector<Shake> shakes(t);
+	vector<bool> cowsInfected(n);
```

Comment:
use `snake_case` to name variables

Reply:
Ok, I will. But you told me in the conversation above to use camelCase.

Reply:
Updated